index.d.ts

```diff
@@ -51,6 +51,35 @@ declare namespace prettyBytes {
 		```
 		*/
 		readonly binary?: boolean;
+
+		/**
+		The minimum number of fraction digits to display.
+
+		@default undefined
+
+		```
+		import prettyBytes = require('pretty-bytes');
+
```

Comment:
Also add an example of not using this option so the user can compare. Same with the other option.

Reply:
Done.